### ISSUE TITLE
Dropout when predicting

### ISSUE THREAD

**Issue by ColdEyeLampr** [jadore801120/attention-is-all-you-need-pytorch#53]
Shouldn't we set dropout prob to 0.0 during prediction?
I notice that in SubLayers.py line 27, the attn_dropout was not set for ScaledDotProductAttention

**Comment by jadore801120** [comment]
Hello @ColdEyeLampr,
Once you switch the model to evaluation mode with `model.eval()`, the dropout will be disabled.
